packages/react/src/composite/SegmentedControl/CHANGELOG.md

```
@@ -2,6 +2,12 @@
 
 The SegmentedControl component is a composite component that allows us to select one or more options from a list. 
 
+## 2025-07-28 `1.1.0`
+
+#### 🎉 New features
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)